source/features/index.md

```diff
@@ -40,25 +40,22 @@ NodeX Agent enables devices to autonomously register the public key to the decen
+Logging data tracks the encryption and decryption status of messaging between devices.
+The data is buffered and sent to NodeX Studio in data format using source ID, destination ID, unique ID for the message, and verification status. If the network of the device is temporarily interrupted, the logging data will be buffered with the maximum value of the pre-configured capacity, and the cached logging data will be sent to NodeX Studio when the network is reopened. NodeX Studio stores the logging data in the server. Basically, the logging data is stored in the server for 2 years, but the storage period depends on the service plan users subscribe to. Users can access the logging data via NodeX Studio API. In the initial configuration, only the admin user can access the logging data. Admin users can change the access control of NodeX Studio. All the logging data can be searched by the logging tags.
 
 ## RoT Extension
 
 Middleware for storing security configurations or secret keys and for accessing RoT functions may sometimes be configured independently. In order to allow NodeX Agent to run in more environments and to serve as a security solution for a variety of use cases, we provide the ability to replace some of the functions built into NodeX Agent with your own. Developers do not need to recompile the entire NodeX Agent. All they need to do is implement the methods according to a simple API specification and create a shared library that can be loaded by NodeX Agent.
 
 ## DID Configuration File
 
-Hub has privately a DID configuration file to ensure device authenticity through message authentication codes by pre-sharing confidential information with the Agent. The DID configuration file in the Hub can be reflected in the json file under the root domain managed by the customer as <a href='https://identity.foundation/.well-known/resources/did-configuration/'>Well Known DID Configuration File</a>.
+Studio has privately a DID configuration file to ensure device authenticity through message authentication codes by pre-sharing confidential information with the Agent. The DID configuration file in the Studio can be reflected in the json file under the root domain managed by the customer as <a href='https://identity.foundation/.well-known/resources/did-configuration/'>Well Known DID Configuration File</a>.
```

Comment:
Since the config would be better covered in the installation procedures, it seems appropriate to remove the entire block.

Reply:
Thank you for your comment. I have removed the block.